WoWsShipBuilder.Web/Utility/ShellSelectionWrapper.cs

```diff
@@ -4,17 +4,28 @@ namespace WoWsShipBuilder.Web.Utility;
 
 public class ShellSelectionWrapper
 {
-    public ShellSelectionWrapper(string shipIndex, Dictionary<string, ArtilleryData> artilleryData)
+    public ShellSelectionWrapper(string shipIndex, Dictionary<string, ArtilleryData> artilleryData, double maxRange)
     {
         ShipIndex = shipIndex;
         ArtilleryDataDictionary = artilleryData;
+        MaxRange = maxRange;
     }
 
     public string ShipIndex { get; }
 
-    public double MaxRange { get; set; } = -1;
+    public double MaxRange { get; set; }
 
     public Dictionary<string, ArtilleryData> ArtilleryDataDictionary { get; }
+
+    protected bool Equals(ShellSelectionWrapper other)
```

Comment:
Override actual Equals method and don't create a new custom method for equality check. If it's not supposed to be public, use a different name.

Reply:
Removed because not used anymore